packages/manager/modules/overthebox/src/overthebox/migration/offers/overTheBox-migration-offers.html

```diff
@@ -131,5 +131,18 @@
                 </table>
             </div>
         </div>
+        <div>
+            <a
+                data-ng-href="{{$ctrl.ABOUT_OTB_SERVICE}}"
+                target="_blank"
+                title="{{::'overthebox_migration_about' | translate}}"
```

Comment:
use data prefix for all if possible

Exemple:
![image](https://github.com/user-attachments/assets/28886503-e16a-4363-874d-52f5eac55a80)